sdk/keyvault/azsecrets/README.md

```diff
@@ -171,15 +171,15 @@ func main() {
 		panic(err)
 	}
 
-	secretName := "mySecretName"
-	secretValue := "mySecretValue"
+	secret := "mySecret"
+	value := "mySecretValue"
```

Comment:
I thought these were going to change to `name` and `secret` (or `value` is also fine) per archboard review. As written, this was confusing.